README.md

```diff
@@ -1,3 +1,7 @@
+<p align="center">
+  <img width="820" height="500" src="https://github.com/dubswalker/gatorgrader/blob/master/gatorgraderlogo.svg">
```

Comment:
This needs to reflect the url of the final image -- where will it be once it is merged to master?

Additionally, you should use markdown, not inline html here, unless there is some special reason.

Reply:
@dubswalker Has there been any progress on this? Before it can be merged to master this must be fixed.

Reply:
Changed it to reflect final url and got rid of incline html. 

Sorry for the delay. If it needs to be changed again let me know! I will be very responsive to this now that my computer works again.